agentlocal/agent_local.go

```diff
@@ -78,19 +92,16 @@ func (als *AgentLocalServer) Status(ctx context.Context, req *agentlocalpb.Statu
 		Latency:      nil, // TODO: Calculate and Add Latency
 	}
 
-	// TODO: Add real AgentsInfo
-	//agentsInfo := &agentlocalpb.AgentInfo{
-	//	AgentId:   "001",
-	//	AgentType: agentpb.Type_MYSQLD_EXPORTER,
-	//	Status:    inventory.AgentStatus_RUNNING,
-	//	Logs:      []string{},
-	//}
+	agentsInfo := als.getAgentsList()
+	if agentsInfo == nil {
+		agentsInfo = []*agentlocalpb.AgentInfo{}
```

Comment:
You actually can't deferential between nil slice and non-nil empty slice at protobuf level. Returning `nil` is fine.